client/views/meet/OngoingCallDuration.js

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
This counter component is not displaying a proper call duration if the page was refreshed by either agent or visitor after joining the call

Reply:
This has been fixed in recent commits